### DISCUSSION TITLE
Github pages site cant be reached

### DISCUSSION THREAD

**Discussion by aedwardz** [github-linguist/linguist#6767] [Q&A]
I deployed a website through github pages with a custom doman and although it says it was deployed and the log shows no errors, the site always says that it cant be reached. the repository is at this link https://github.com/aedwardz/aedwardz.github.io

**Comment by lildude** [comment]
You're asking in the wrong place. Try https://github.com/orgs/community/discussions/categories/pages or GitHub Support.
